server/app/services/mongo_pubsub.rb

```diff
@@ -93,7 +101,7 @@ def unsubscribe(subscription)
   def publish(channel, data)
     self.collection.insert_one(
       channel: channel,
-      data: data,
+      data: BSON::Binary.new(Marshal::dump(data)),
```

Comment:
That should work too. 

In the first try encountered some issues with msgpack UTF-8 in-compliance, tried with plain Ruby `Marshal` (which has the same issue btw) and left it there. :)

So no real reason, can change this to use msgpack if wanted?